tools/storm/scripts/build_extension_images/build_extension_images.go

```diff
@@ -58,6 +58,7 @@ func (s *BuildExtensionImagesScript) Run() error {
 
 func buildImage(extType string, numClones int) error {
 	for i := 1; i <= numClones; i++ {
+		extName := fmt.Sprintf("test-%s-%d", extType, i)
```

Comment:
does this mean that the extensions are no longer versions of the same extension?